asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -100,8 +102,11 @@ def __init__(
         self.output_paths = []
         # make sure all paths exist, otherwise skip
         for pose, path in zip(poses, output_paths):
-            if pose and Path(pose).exists():
-                mol = load_openeye_sdf(str(pose))
+            if pose:
+                if isinstance(pose, oechem.OEMolBase):
+                    mol = pose.CreateCopy()
+                else:
+                    mol = load_openeye_sdf(str(pose))
```

Comment:
1) why not use the Ligand schema here?
2) if `poses` can be an `oechem.OEMolBase`, that should be reflected in the type hints above? otherwise I think the `if isinstance` method should be removed

Reply:
great point, ill fix.

Reply:
The reason to not use ligand is TBH because I am planning on completely redoing in V2 as noted in #617, for now I have added in the correct type hints.